client/src/components/AllTabs/FirstTab.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+function FirstTab() {
```

Comment:
Not sure where you use FirstTab/SecondTab? Do we need these files?

Reply:
Oh this was for testing thanks I will remove!